html-forms/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="/my-handling-form-page" method="post">
+      <fieldset>
+        <legend>Fill-in Information</legend>
+
+        <p>
+          <label for="mail">Email:</label>
+          <input type="email" id="mail" name="user_email" />
+        </p>
+        <p>
+          <label for="dropdown">Why is coding cool?</label>
+        </p>
+        <p>
+          <select>
```

Comment:
The select should have an `id="dropdown"` so it's linked to the `label` on line 19